### DISCUSSION TITLE
CODE : How enable track change on CODE by default

### DISCUSSION THREAD

**Discussion by franckblack** [CollaboraOnline/online#10629] [Q&A]
Hi,

I install on my server a version of Collabora online for developpment and i want to know how activate by default track change when i open a word on a iframe

Thanks for your help

Best regards

**Comment by vmiklos** [comment]
Sound like you would like to tweak defaults for how Writer in core works; normally coolkitconfig.xcu file makes that possible, you can add custom settings there. That being said, I'm not aware of a setting to enable track changes by default, unless you create a template that has this, and all new documents are based on that template -- it's a per-document setting by default.

You may want to file a feature request to have such a setting.

In general, best to ask such questions at https://forum.collaboraonline.com/, GH Discussions isn't used much, unfortunately.
